File: 7kue/Leap Years/solve-1.ts
function isLeap(year: number):boolean {
  return year % 4 === 0 && year % 100 !== 0 || year % 400 === 0
}

console.log(isLeap(2000))  // True
console.log(isLeap(1604))  // True
console.log(isLeap(1600))  // True
console.log(isLeap(2015))  // False

File: 7kue/Leap Years/solve-1.py
def is_leap_year(year):
    return year % 4 == 0 and year % 100 != 0 or year % 400 == 0 


print(is_leap_year(2000))  # True
print(is_leap_year(1604))  # True
print(is_leap_year(1600))  # True
print(is_leap_year(2015))  # False
